verifier/src/report_body.rs

```diff
@@ -811,10 +856,14 @@ mod test {
 
         let mr_signer_verifier = MrSignerVerifier::new(mr_signer, product_id, isv_svn);
 
-        assert_eq!(
-            mr_signer_verifier.verify(&report_body).is_ok().unwrap_u8(),
-            1
-        );
+        let verification = mr_signer_verifier.verify(&report_body);
+        assert_eq!(verification.is_ok().unwrap_u8(), 1);
+        let expected = r#"
+            - [x] MrSigner:
+              - [x] The MRSIGNER key hash was MrSigner(sgx_measurement_t { m: [48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79] })
+              - [x] The ISV product ID was IsvProductId(144)
+              - [x] The ISV SVN was IsvSvn(145)"#;
+        assert_eq!(format!("\n{verification}"), textwrap::dedent(expected));
```

Comment:
Generally I lean away from not asserting multiple things in unit tests, but I felt with the setup required here it was best to verify both items.

One could unit tests the `MrSignerVerificationMetadata` separately, but that doesn't assure that `MrSignerVerifier::verify()` returns that type correctly.